src/server/server.ts

```diff
@@ -47,6 +47,15 @@ function isOfficeHost(host: string) {
     return officeHosts.indexOf(host) >= 0;
 }
 
+let assetPaths = null;
+function getAssetPaths() {
+    if (assetPaths === null) {
+        let data = fs.readFileSync(path.resolve(__dirname, 'assets.json'));
+        assetPaths = JSON.parse(data.toString());
+    }
+
+    return assetPaths;
+}
 
```

Comment:
Logic behind having it as a function was that if, for some reason, the file read fails, then reloading the page would call the function again, triggering a re-read. Having a `const` or something similar at the top would not accomplish that, I believe. But am open to other suggestions.